persistence/patch-sql-clouddriver.yml

```diff
@@ -41,17 +41,23 @@ spec:
             tasks:
               user: clouddriver_service
               password: encrypted:k8s!n:spin-secrets!k:mysqlCdPassword  # (Secret). Depending on db auth and how spinnaker secrets are managed
-              jdbcUrl: jdbc:mysql://mysql:3306/clouddriver
+              jdbcUrl: jdbc:mysql://mysql:3306/clouddriver?useSSL=false&useUnicode=true&characterEncoding=utf8 # useUnicode and utf8 makes sure the database can store an emoji if one is ever added to the database
           migration:
             user: clouddriver_migrate
             password: encrypted:k8s!n:spin-secrets!k:mysqlCdMigratePassword  # (Secret). Depending on db auth and how spinnaker secrets are managed
-            jdbcUrl: jdbc:mysql://mysql:3306/clouddriver
-
-        redis:
+            jdbcUrl: jdbc:mysql://mysql:3306/clouddriver?useSSL=false&useUnicode=true&characterEncoding=utf8 # useUnicode and utf8 makes sure the database can store an emoji if one is ever added to the database
+        
+        dualTaskRepository: # Maintaining Task Repository Availability While Migrating from Redis to SQL in production https://spinnaker.io/setup/productionize/persistence/clouddriver-sql/#maintaining-task-repository-availability-while-migrating-from-redis-to-sql
           enabled: false
+          primaryClass: com.netflix.spinnaker.clouddriver.sql.SqlTaskRepository
+          previousClass: com.netflix.spinnaker.clouddriver.data.task.jedis.RedisTaskRepository
+        
+        redis:
+          enabled: true
+          connection: redis://redis:6379 # only required if using externally managed redis AND Google CloudSQL or Amazon Managed RDS SQL
```

Comment:
I think this `connection` setting needs to be either: 
* Commented out by default, so that an external redis is not required out of the box when using this config
* Set to `redis://spin-redis:6379`, so that it uses the already existing redis (not recommended for production)

Reply:
agreed, will fix